srcpkgs/lsof/template

```diff
@@ -1,46 +1,43 @@
 # Template file for 'lsof'
 pkgname=lsof
-version=4.94.0
+version=4.95.0
 revision=1
 build_style=configure
 configure_script="./Configure"
 configure_args="-n linux"
-hostmakedepends="perl which"
+make_check_target="all"
+make_check_args="-C tests"
+hostmakedepends="perl which groff"
 short_desc="LiSt Open Files"
-maintainer="Orphaned <orphan@voidlinux.org>"
+maintainer="Subhaditya Nath <sn03.general@gmail.com>"
 license="custom:lsof"
-homepage="https://people.freebsd.org/~abe/"
-distfiles="https://fossies.org/linux/misc/lsof-${version}.tar.gz"
-checksum=a9865eeb581c3abaac7426962ddb112ecfd86a5ae93086eb4581ce100f8fa8f4
+homepage="https://github.com/lsof-org/lsof"
+changelog="https://raw.githubusercontent.com/lsof-org/lsof/master/00DIST"
+distfiles="https://github.com/lsof-org/lsof/archive/refs/tags/${version}.tar.gz"
+checksum=8ff4c77736cc7d9556da9e2c7614cc4292a12f1979f20bd520d3c6f64b66a4d7
+
+post_extract() {
+	rm tests/CkTestDB
+	ln -s /bin/true tests/CkTestDB
```

Comment:
Would you mind adding a comment for this one?

Reply:
done.